parts/k8s/kubernetesmasterresources.t

```diff
@@ -657,7 +657,7 @@
               "createOption": "Empty"
               ,"diskSizeGB": "[variables('etcdDiskSizeGB')]"
               ,"lun": 0
-              ,"name": "[concat(variables('masterVMNamePrefix'), copyIndex(variables('masterOffset')),'-etcddisk')]"
+              ,"name": "[concat(variables('masterVMNamePrefix'), copyIndex(variables('masterOffset')),'_EtcdDisk')]"
```

Comment:
I think all other resource names are lower case (e.g. _clustername-master-availabilityset_). Are you sure about deviating from that convention here?